src/types.ts

```diff
@@ -27,13 +27,15 @@ export interface ListrContextPublish {
   [dnpName: string]: ListrContextPublishItem;
 }
 
-interface PublishVariantsMapEntry {
+interface PackageToPublishProps {
   // Manifest-related
   manifest: Manifest;
   manifestFormat: ManifestFormat;
 }
 
-export interface BuildVariantsMapEntry extends PublishVariantsMapEntry {
+export interface PackageToBuildProps extends PackageToPublishProps {
+  variant: string | null;
```

Comment:
What are your intentions allowing `null` if null means something either add a comment or find a way to define such intention with a boolean

Reply:
also, what means exactly `variant`

Reply:
I added a comment. `variant` could be `mainnet`, `testnet`, `holesky`... the value that makes one package differ from another in the same repo. It must be equal to the directory where that variant files are defined. For example, if you want to build the `holesky` and `mainnet` variants of a package, you would run:

`dappnodesdk build --variants-dir "./package_variants" --variants "mainnet,holesky"

And your repo must include these files:

```
.
├── avatar.png
├── dappnode_package.json
├── docker-compose.yml
...
├── package_variants
│   ├── holesky
│   │   ├── dappnode_package.json
│   │   └── docker-compose.yml
│   └── mainnet
│       ├── dappnode_package.json
│       └── docker-compose.yml
└── README.md
```